### DISCUSSION TITLE
【灌水】是我不懂品咖啡吗

### DISCUSSION THREAD

**Discussion by gaojiaobei** [share121/inter-knot#896] [其他]
最近去了一趟六分街出差，问了一下路人有什么一定要去的地方，路人向我推荐了汀曼大师的咖啡店，说喝上一杯甚至玩VR都有劲了，不喝这一杯新艾利都白来。
半信半疑的尝试了一下，果然，这种邪乎的名声几乎都是假的。旧都都陷落多少年了，还在用这种传统的手磨方式，口感味道没感觉比速溶的好上多少，出餐效率倒是慢了许多，等上这么一会，喝咖啡的心情都没了。至于有没有对精神的加成，我没有VR，就用手游代替尝试了一下，感觉没什么效果，不过以讹传讹罢了。
看到这拖累咖啡行业发展的糟粕，我不由得想起了最近崛起的鸠巢速溶咖啡，热水冲泡半分钟即可饮用，口感香醇浓厚，提神效果也很好。
更重要的是，最近鸠巢搞优惠，点击下方链接，200丁尼即可抢到三个月的量，别再犹豫，快来下单吧。
优惠链接：https://www.bilibili.com/video/BV1im42137GG/?spm_id_from=333.999.0.0&vd_source=35c0d1c8c57b5250858c36aa5e065dca

**Comment by ran-mitake-ag** [comment]
别的我不说，但是月壤摩咔是真好喝。_不要喝淡的！！_

**Comment by Lephaethon** [comment]
绳网最近怎么这么多广告商啊

**Comment by warning-Kirov-reporting** [comment]
绳网名气大了就会有奇奇怪怪的店铺来的。
